gpkit/shortcuts.py

```diff
@@ -1,12 +1,15 @@
 from .variables import Variable
 from .variables import VectorVariable
 from .variables import ArrayVariable
-from .geometric_program import GeometricProgram
-from .signomial_program import SignomialProgram
+from .model import Model
 
 Var = Variable
 Vec = VectorVariable
 Arr = ArrayVariable
+Model = Model
 
-GP = GeometricProgram
-SP = SignomialProgram
+def GP(*args, **kwargs):
+    raise Exception("'Model' has replaced 'GP'. Please update your code!")
+
+def SP(*args, **kwargs):
+    raise Exception("'Model' has replaced 'GP'. Please update your code!")
```

Comment:
I think this is too harsh. Is the API backward compatible in any way? e.g. can I call `Model` with the same args as `GP`? if so why not set `GP=Model` and print a warning instead of raising?